peekingduck/pipeline/nodes/draw/brightness.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Adjusts brightness of an incoming image
+"""
+
+
+from typing import Any, Dict
+from PIL import Image, ImageEnhance
```

Comment:
As discussed, it's observed that Pillow decreases FPS by ~10% and introduces a new package. Let's stick with OpenCV calculations for brightness adjustment

Reply:
Found a particular opencv image manipulation method for Alpha (contrast) and beta (brightness) of images that follows `result = alpha * image(x,y) + beta`. Implemented contrast and brightness using this new method. This is much faster and does not affect fps noticeably.

Reply:
Nice!